src/views/Pools/components/PoolCard/Modals/StakeModal.tsx

```diff
@@ -43,7 +43,13 @@ const StakeModal: React.FC<StakeModalProps> = ({
   const [stakeAmount, setStakeAmount] = useState('')
   const [hasReachedStakeLimit, setHasReachedStakedLimit] = useState(false)
   const [percent, setPercent] = useState(0)
-  const calculatedStakingLimit = stakingTokenBalance.gt(stakingLimit) ? stakingLimit : stakingTokenBalance
+  const getCalculatedStakingLimit = () => {
+    if (isRemovingStake) {
+      return userData.stakedBalance
```

Comment:
Currently there's no specific handling for unstaking from pools, this sets the limit to the total user's stakedBalance when unstaking